gax-java/gax/src/main/java/com/google/api/gax/longrunning/OperationTimedPollAlgorithm.java

```diff
@@ -35,14 +35,23 @@
 import com.google.api.gax.retrying.ExponentialRetryAlgorithm;
 import com.google.api.gax.retrying.RetrySettings;
 import com.google.api.gax.retrying.TimedAttemptSettings;
+import com.google.common.annotations.VisibleForTesting;
 import java.util.concurrent.CancellationException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 /**
  * Operation timed polling algorithm, which uses exponential backoff factor for determining when the
  * next polling operation should be executed. If the polling exceeds the total timeout this
  * algorithm cancels polling.
  */
 public class OperationTimedPollAlgorithm extends ExponentialRetryAlgorithm {
+
+  public static Logger LOGGER = Logger.getLogger(OperationTimedPollAlgorithm.class.getName());
```

Comment:
Why the logger has to be `public`?

Reply:
This is so it can be accessed by the test in order to add a log handler. @lqiu96, maybe we can fall back to `@VisibleForTesting` because it is... visible only for testing?

Reply:
Ah, right. Let's try the the default package scope, which I believe should still be accessible. If not, then we can add the annotation.

Reply:
There're some suggestions made by Quality Gate: https://sonarcloud.io/project/issues?id=googleapis_gapic-generator-java&pullRequest=2236&resolved=false&types=CODE_SMELL